src/points/slice.ts

```diff
@@ -71,6 +80,16 @@ const slice = createSlice({
     getPointsConfigRetry: (state) => ({
       ...state,
     }),
+    pendingPointsEventAdded: (state, action: PayloadAction<PendingPointsEvent>) => ({
+      ...state,
+      pendingPointsEvents: [...state.pendingPointsEvents, action.payload],
+    }),
+    pendingPointsEventRemoved: (state, action: PayloadAction<Pick<PendingPointsEvent, 'id'>>) => ({
```

Comment:
good point, thanks! changed it to:
* `sendPointsEventStarted` (similar to `getHistoryStarted` and ‘getPointsConfigStarted` above)
* `pointsEventProcessed` (because this one is dispatched not only when the points event is sent, but also when the expired points event is discarded)